### PR TITLE
Version Change Only - Snips 0.60.1

### PR DESCRIPTION
**Pull request by tschmidty69**
Updated to latest snips 0.60.1

### CODE REVIEW

**Comment by pvizeli** [comment]
there is no arm image?

**Comment by tschmidty69** [comment]
Ugh, didn't notice that, misplaced my sd card with my docker build environment so was winging it. I could build one but myself but we should just hold this until they post theirs.

**Comment by pvizeli** [comment]
We can also use our Ubuntu base images and build itself with the packages...

**Comment by tschmidty69** [comment]
I sent them a message on discord. I can definitely build an image, might be worth it since I dont think they will respond until monday at the earliest

**Comment by pvizeli** [comment]
I'm not sure, but it looks like that snips don't support his docker anymore and favor to install it with apt-get and his repository.

I'm fine with both, as followup docker build or as own image.

**Comment by tschmidty69** [comment]
I believe there was a bug with arm on pi zero they were trying to fix. I'll update my images on dockerhub and update the pr in a bit.

**Comment by pvizeli** [comment]
That images are from you?

https://snips.gitbook.io/documentation/advanced-configuration/advanced-solutions#docker

**Comment by pvizeli** [comment]
Any news?

File: snips/CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
 # Changelog
 
+## 5
+- Updated to snisp 0.60.1
```
Comment (tjorim) [inline snips/CHANGELOG.md]:
```suggestion
- Updated to snips 0.60.1
```

**Comment by tjorim** [comment]
Using other base images might be better since we have no control over the others, can't seem to find the Dockerfile they use either.
There was a user on a raspi 3b+ with 64 bit HassOS (aarch64) that wondered why he couldn't install this add-on, but we simply don't have a base image for that arch now.

**Comment by cheelio** [comment]
I think this image is based on this base image: https://github.com/syntithenai/opensnips/blob/master/docker-images/snips/Dockerfile (see how this run.sh points to the other image's entrypoint (/opt/snips/snips-entrypoint.sh)

**Comment by tschmidty69** [comment]
Yeah, they keep saying the image is due 'any day now...' but they keep saying that. 

The issue comes up that someone has to maintain that image though which is why we have tried to use the official one. Mainly because they release new versions when they update their ASR and console versions but in this case they ran into a bug with the ARM version. 

@cheelio yes, that is what the hassio addon does as well.

**Comment by tschmidty69** [comment]
Give me a bit and I will repoint to my docker repository for this release and move the default assistant stuff to a separate repo

**Comment by adamotte** [comment]
Can we hope an update soon ?

**Comment by homeassistant** [comment]
Hello @tschmidty69,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`395ed66904c5d60170e486107f5d51b841e916e8`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;hassio-addons&#x2F;pull&#x2F;458&#x2F;commits&#x2F;395ed66904c5d60170e486107f5d51b841e916e8) - This commit has something that looks like an email address (root@snips-default). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

**Comment by homeassistant** [comment]
Hello @tschmidty69,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`395ed66904c5d60170e486107f5d51b841e916e8`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;hassio-addons&#x2F;pull&#x2F;458&#x2F;commits&#x2F;395ed66904c5d60170e486107f5d51b841e916e8) - This commit has something that looks like an email address (root@snips-default). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

**Comment by pvizeli** [comment]
two question:

1) I upload the assistant files to S3 on the start of this PR. Has this changed now?
2) You provide now own docker base images, that's fine. But would make sense to integrate your Dockerfile into that from us?

**Comment by tschmidty69** [comment]
I think you accidentally added the s3 stuff in a different commit since it is already there on the master branch.

I built this based on the snipsdocker image. I could have rebuilt it all but figured it would be best to get it from them. Essentially all I did was upgrade the packages so we could base it on this image going forward and add that 'apt-get upgrade snips-platform' to the Dockerfile and perhaps copy these images to the official hassio-addons docker hub. Then just bumping the version and rebuilding the image should do what we need and we could address any conflicts down the road if there are major updates.

File: snips/Dockerfile [hunk 2]
```diff
@@ -8,6 +8,7 @@ COPY customtts.sh /usr/bin
 
 RUN chmod a+x /run.sh \
     && chmod a+x /usr/bin/customtts.sh \
+    && echo nameserver 1.1.1.1 > /etc/resolv.conf \
```
Comment (pvizeli) [inline snips/Dockerfile]: Remove this, docker start modify also the resolv

**Comment by Romkabouter** [comment]
After updating the addon an downloading a new assistant from the console, I get this message:

ERROR:snips_nlu       : Unable to read file '/usr/share/snips/assistant/nlu_engine/nlu_engine.json'
 -> caused by: Expected model version 0.16.0 but found 0.17.

Anyone else?

**Comment by tschmidty69** [comment]
Hmm, trying to take a look but I can no longer connect my mqtt broker on localhost:1883. Is yours picking up version 5?

**Comment by Romkabouter** [comment]
Yes, I have first uninstalled version 4 to verify.
So I installed addon version 5 and download a new assistant. Assistant says version 17 in assistant.json
I will try that assistant in my other snips (used for testing, stand alone) as well to be sure

**Comment by brainexe** [comment]
here are some other reports with the same error:
https://github.com/home-assistant/hassio-addons/issues/487#issuecomment-447026347

I also deinstalled the snips 4 and reinstalled the new version 5.0 + used a new assistant.zip...without success

**Comment by Heronimonimo** [comment]
> Hmm, trying to take a look but I can no longer connect my mqtt broker on localhost:1883. Is yours picking up version 5?

I have the same problem, nlu version works for me but problems with connecting to the mqtt broker

**Pull request merged**